site/src/components/FormFooter/FormFooter.tsx

```diff
@@ -45,6 +47,7 @@ export const FormFooter: FC<React.PropsWithChildren<FormFooterProps>> = ({
         variant="contained"
         color="primary"
         type="submit"
+        disabled={submitDisabled}
```

Comment:
What's the reasoning for adding this?

Reply:
This lets us control whether the submit button is disabled or not, so for instance if we fail to fetch the quotas there will be an error and the form should not be submittable.

Reply:
Got it! Thanks for clarifying. Sounds like a nice UX improvement 👏🏼